setup/www/resources/config/benchmarking.nodejs.org

```diff
@@ -0,0 +1,38 @@
+server {
+    listen *:80;
+    server_name benchmarking.nodejs.org;
+
+    return 301 https://benchmarking.nodejs.org$request_uri;
+}
+
+server {
+    listen [::]:443 ssl http2;
+    listen *:443 ssl http2;
+    server_name benchmarking.nodejs.org;
+
+    ssl_certificate ssl/nodejs_chained.crt;
+    ssl_certificate_key ssl/nodejs.key;
+    ssl_trusted_certificate ssl/nodejs_chained.crt;
+
+    keepalive_timeout 60;
+    server_tokens off;
+
+    resolver 8.8.4.4 8.8.8.8 valid=300s;
+    resolver_timeout 10s;
+
+    add_header Strict-Transport-Security max-age=63072000;
```

Comment:
might be a bit too complex for this, we have subdomains that we want to still serve on port 80, including unencrypted.nodejs.org